src/scrcmd_c.c

```diff
@@ -4147,23 +4148,23 @@ BOOL ScrCmd_595(ScriptContext *ctx) {
 }
 
 BOOL ScrCmd_627(ScriptContext *ctx) {
-    struct UnkStruct_ScrCmd627 **p_work = FieldSysGetAttrAddr(ctx->fieldSystem, SCRIPTENV_RUNNING_APP_DATA);
+    struct FrontierLaunchParam **p_work = FieldSysGetAttrAddr(ctx->fieldSystem, SCRIPTENV_RUNNING_APP_DATA);
     u8 r6 = ScriptReadByte(ctx);
-    struct UnkStruct_ScrCmd627 *work = AllocFromHeapAtEnd(HEAP_ID_FIELD, sizeof(struct UnkStruct_ScrCmd627));
-    MI_CpuClear8(work, sizeof(struct UnkStruct_ScrCmd627));
+    struct FrontierLaunchParam *work = AllocFromHeapAtEnd(HEAP_ID_FIELD, sizeof(FrontierLaunchParam));
```

Comment:
no struct

Reply:
Agree. Typedef the struct and use the typedef'd name.

Reply:
Looks like this was an artifact; tons more in scrcmd_c since it's an older file that we can get to as structs get named

Reply:
understandable, since it begins with S, it's low on my cleanup list, but I'll get there eventually